WordPress/Classes/Utility/Analytics/MixpanelProxy.m

```diff
@@ -0,0 +1,97 @@
+#import "MixpanelProxy.h"
+
+#import <Mixpanel/Mixpanel.h>
+
+
+@interface MixpanelProxy ()
+
+@property (nonatomic, strong) dispatch_queue_t superPropertiesQueue;
+
+@end
+
+@implementation MixpanelProxy
+
+- (instancetype)init
+{
+    self = [super init];
+    if (self) {
+        _superPropertiesQueue = dispatch_queue_create("org.wordpress.analytics.mixpanelproxy", DISPATCH_QUEUE_SERIAL);
```

Comment:
I was going to use `NSLock` for this but then I realized I could simplify things and just use the serial queue ;).